.github/workflows/bugzilla_redmine.yml

```diff
@@ -0,0 +1,37 @@
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
+# bootstrap.py to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+---
+name: Bugzilla Redmine
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 */4 * * *'
+jobs:
+  redmine_bugzilla:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Bugzilla automation script
+        env:
+          REDMINE_BUGZILLA_CONF: ${{ secrets.REDMINE_BUGZILLA_CONF }}
```

Comment:
Need secret!

Reply:
What format does this need to be in?

Reply:
INI format:
https://github.com/pulp/pulp-ci/blob/master/ci/redmine_bugzilla.py#L25-L38

Reply:
The secret for the redmine key is `REDMINE_API_KEY` and it'll be `BUGZILLA_USERNAME` and `BUGZILLA_PASSWORD` for the bz credentials.

Reply:
So I've set `BUGZILLA_API_KEY` as a secret on this repo. They're disabling use of username+password in favor of api keys. You should be able to do this:

```python
RHBugzilla(f"{BUGZILLA_URL}/xmlrpc.cgi", api_key=api_key)
```